erpnext_documentation/www/docs/user/manual/en/human-resources/attendance-status.md

```diff
@@ -0,0 +1,43 @@
+<!-- add-breadcrumbs -->
+# Attendance Status
+
+Attendance Status the status of an employee, Which represent That wether Employee is Present, Absent, or is on leave on particular day.
+
+In ERPNext You can create multiple Status for Attendance. Suppose, Employee is on visit to other place or working remotely in that case we allow you to create custom status for tracking Attendance.
+
+To access Attendance Status, go to:
+
+> Home > Human Resources > Attendance
+
+## 1. How to Create Attendance status:
+
+1. Go to the Attendance Status list, click on New.
+1. Name your Attendance Status.
+1. Write Abbreviation for your status. Which is used in Monthly Attendance Sheet.
+1. Click on save.
+
+> Note: if None of the property Is Half Day, Is Leave or Is Present is checked status will be treated as Absent
+
+<img class="screenshot" alt="Attendance" src="{{docs_base_url}}/assets/img/human-resources/attendance-status.png">
+
+Below are detailed explanation of all checkbox in Attendance Status.
+
+* **IS Half Day:** If Checked, The Attendance Status will represent Half Day Status During Attendance Creation.
+
+* **IS Leave:** In ensure that Attendance Status will treated as leave Status.
```

Comment:
```suggestion
* **Is Leave:** If checked, the Attendance Status will be set as Leave during the attendance creation.
```

Reply:
There is nothing to be set. Attendance Status is a doctype where users can define their own attendance status.